src/ui/MediumVideoPreview/MovieDetails.tsx

```diff
@@ -17,15 +17,16 @@ const MovieDetails = ({
   buy,
   rent
 }: MovieDetailsProps): ReactElement => {
-  // determine what buy/rent text is
   return (
     <div className="movie-details">
       <h1>{shortsTitle}</h1>
       <div>
-        {genre} - {year}
+        {genre} • {year}
       </div>
       <div>
-        <span>show buy/rent text here</span>
+        <span className="purchase-options">
+          {rent === true ? 'Buy or Rent' : 'buy'}
```

Comment:
this is wrong logic.

1. Buy or Rent should show only if `rent` and `buy` are true.
2. Rent should show when `rent` is true
3. Buy should show when `buy` is true

Right now Buy or Rent will show when `rent` is true, otherwise Buy will show